cmd/ncproxy/run.go

```diff
@@ -89,6 +107,7 @@ func run(clicontext *cli.Context) error {
 	var (
 		configPath    = clicontext.GlobalString("config")
 		logDir        = clicontext.GlobalString("log-directory")
+		dbPath        = clicontext.GlobalString("log-directory")
```

Comment:
Should be `database-path` instead of `log-directory`?

Reply:
Good catch :) fixed